frameworks/helloworld/src/main/dist/examples/marathon_constraint.yml

```diff
@@ -0,0 +1,43 @@
+name: "hello-world"
+pods:
+  hello:
+    count: {{HELLO_COUNT}}
+    placement: {{HELLO_PLACEMENT}}
```

Comment:
`svc.yml` doesn't have a plan strategy exposed, and I want the tasks to deploy in parallel while keeping the phases serial. Otherwise the tests will take over an hour. Also, I'd prefer to guard against any future changes to `svc.yml` and have something that acts more like a test fixture tied to this suite of tests.